src/Config/Main.hs

```diff
@@ -15,6 +15,7 @@ import qualified Data.Text as T
 import System.Environment (lookupEnv)
 import System.Directory (getHomeDirectory)
 import Data.Maybe (fromMaybe)
+import GHC.Exts (fromList)
```

Comment:
Can we import `Data.List.NonEmpty.fromList` instead?

Reply:
Yep